src/components/Lenkepanel/LenkepanelWrapper.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { Element, Undertekst, Undertittel } from 'nav-frontend-typografi';
+import { LenkepanelBase } from 'nav-frontend-lenkepanel';
+
+import './LenkepanelWrapper.less';
+import { Periode, Arbeidsgiver, Sykmelding } from '../../types/sykmeldingTypes';
+import dayjs from 'dayjs';
+
+interface LenkepanelProps {
+    lenke: string;
+    tittel: string;
+    sykmeldinger: Sykmelding | Sykmelding[];
+    tekst: string;
+    svg: string;
+}
+
+const beregnPeriodelengde = (fom: Date, tom: Date): number => dayjs(tom).diff(dayjs(fom), 'day');
+const tilLesbarPeriodeMedGraderingOgArbeidsgiver = (
```

Comment:
Tenker det er greit å skille ut denne til en egen fil ved siden av komponenten, og at det skrives noen tester på den.

Reply:
enig der. tenker å refakturere den her litt for å gjøre den mer generell (kommenter i #12)